pkgs/development/libraries/ncurses/default.nix

```diff
@@ -46,6 +46,9 @@ stdenv.mkDerivation rec {
   buildInputs = lib.optional (mouseSupport && stdenv.isLinux) gpm;
 
   preConfigure = ''
+    # These paths end up in the default lookup chain.
+    export TERMINFO_DIRS=/etc/terminfo
+
     export PKG_CONFIG_LIBDIR="$dev/lib/pkgconfig"
```

Comment:
Thank you @4z3 for providing references. I was unable to find any.
Now I really refuse to make a change according to an undocumented implementation of an unpublished (and not freely available) standard ;-).
I also agree that $out should never contain spaces, but again, look at the length of this discussion just to agree that not quoting is correct, while everybody agrees easily that quoting is correct.